include/dos_system.h

```diff
@@ -36,6 +36,10 @@
 #define DOS_TEMPSIZE 1024
 #define DOS_MFNLENGTH 8
 #define DOS_EXTLENGTH 3
+
+constexpr uint8_t DOS_MFNLENGTH_STRING = DOS_MFNLENGTH + 1;
+constexpr uint8_t DOS_EXTLENGTH_STRING = DOS_EXTLENGTH + 1;
```

Comment:
Don't we want to use lower_camel_case consistently for constants from now on?

Reply:
Indeed; I think these additions can be avoided entirely if we move to std::string handling in the function.
`